charts/opentelemetry-demo/examples/bring-your-own-observability/rendered/component.yaml

```diff
@@ -5,130 +5,119 @@ kind: Service
 metadata:
   name: example-adservice
   labels:
-    helm.sh/chart: opentelemetry-demo-0.20.2
-    app.kubernetes.io/name: example
-    app.kubernetes.io/instance: example
-    app.kubernetes.io/component: adservice
+    helm.sh/chart: opentelemetry-demo-0.21.0
+    
+    opentelemetry.io/name: example-adservice
```

Comment:
since `/instance` and `/component` were moved to the new helper the service labels changed.  I think the service template needs updated like the deployment template was.

Reply:
Thanks @TylerHelmuth .
it's fixed in latest commit as below ( change one line in `charts/opentelemetry-demo/templates/_helpers.tpl`)

```
     {{/*
     Common labels
     */}}
     {{- define "otel-demo.labels" -}}
     ...
     {{ include "otel-demo.selectorLabels" . }}
+  {{ include "otel-demo.workloadLabels" . }}
     ...
     {{- end }}
```